org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -12,7 +12,7 @@ $button-border-radius: 2px;
     opacity: 1 !important;
     font-size: 1em !important;
 
-    &:not(.crm-form-submit):not(.crm-close-accordion):not(.crm-clear-link) {
+    &:not(.crm-form-submit):not(.crm-close-accordion):not(.crm-clear-link):not([href^="/index.php?q=civicrm/vacancy/apply&reset=1&id="]) {
```

Comment:
This is way over-specific. What are you trying to accomplish here? Isn't there a better selector to do it? In any case i think this selector in general deserves a comment to explain why it has this `:not()`s in the first place